tpl/bootstrap.php

```diff
@@ -55,7 +55,7 @@
 endif;
 ?>
 		<script type="text/javascript" data-cfasync="false" src="js/zlib-1.2.11.js" integrity="sha512-Yey/0yoaVmSbqMEyyff3DIu8kCPwpHvHf7tY1AuZ1lrX9NPCMg87PwzngMi+VNbe4ilCApmePeuKT869RTcyCQ==" crossorigin="anonymous"></script>
-		<script type="text/javascript" data-cfasync="false" src="js/base-x-3.0.7.js" integrity="sha512-/Bi1AJIP0TtxEB+Jh6Hk809H1G7vn4iJV80qagslf0+Hm0UjUi1s3qNrn1kZULjzUYuaf6ck0ndLGJ7MxWLmgQ==" crossorigin="anonymous"></script>
+		<script type="text/javascript" data-cfasync="false" src="js/base-x-4.0.0.js" integrity="sha512-nNPg5IGCwwrveZ8cA/yMGr5HiRS5Ps2H+s0J/mKTPjCPWUgFGGw7M5nqdnPD3VsRwCVysUh3Y8OWjeSKGkEQJQ==" crossorigin="anonymous"></script>
```

Comment:
Hmm, checked https://unpkg.com/base-x@4.0.0/src/index.js and got `sha512-qmPD4cjowEPb9yj7VdDOxTvk3izuo4YmMJSDR3pujPVEtXlYYSRns+J6lELoNHiiXWz2x8LdAQ/k3iy75adnCw==`

I manually checked the files and looked at the source here: https://github.com/cryptocoinjs/base-x/blob/master/src/index.js

And you really sure you've got v4.0.0?


Ahhh... it is some other non CommonModule thingy version, but more ECMAScript module, apparently? Where did you get that from?

Anyway, the local files looks benign enough so fair enough.

Reply:
I always have to convert their nodejs-only syntax in something we can use in a browser. So it's modified by me.